torch/_inductor/codecache.py

```diff
@@ -1263,11 +1263,12 @@ def _check_can_cache(gm: torch.fx.GraphModule) -> None:
             log.debug("fx graph cache no shape env")
             raise BypassFxGraphCache("No shape env")
 
-        # HigherOrderOperators should be handled on a case-by-case basis.
-        # Currently, we just skip caching if we have any.
-        # We also skip if there are any torchbind objects.
+        # We skip caching if there are any torchbind objects.
         for node in gm.graph.nodes:
-            if isinstance(node.target, torch._ops.HigherOrderOperator):
+            if (
+                isinstance(node.target, torch._ops.HigherOrderOperator)
```

Comment:
Do we recurse into the subgraph of the HOP as a part of the cache key?

Reply:
We pickle the `torch.fx.GraphModule`, so yes

Reply:
How does this work exactly ? Would you please print an example ?